### ISSUE TITLE
Footer text missing ARIA landmark

### ISSUE THREAD

**Issue by rjkati** [samvera/hyrax#6838]
### Descriptive summary
On dev.nurax.samvera.org (Hyrax 5.0.1), the footer text on public facing pages is not included in an ARIA landmark

### Steps to reproduce the behavior in User Interface (UI)

1. In Chrome, download the Site Improve browser extension
2. Go to https://dev.nurax.samvera.org/ and run the Site Improve browser extension
![aria_footer](https://github.com/samvera/hyrax/assets/32644254/5f8d1f1a-753f-4f13-b1ab-8e0afb1151ec)

**Comment by aahurford** [comment]
tested and confirmed issue.

**Comment by jlhardes** [comment]
@rjkati - I am not seeing this SiteImprove error on dev.nurax or locally. This might have been addressed by an upgrade of something  along the way?

**Comment by christhepianist** [comment]
Confirmed that this is no longer an issue.
